OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+          "text": "Anesthesiology Preoperative evaluation and management note"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "content": [
+          {
+            "attachment": {
+              "contentType": "application/pdf",
+              "data": "bHVuZzZtb3d0",
+              "title": "Evaluation note"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:910e1ca3-c2e6-4e54-b76a-a12c719aa8a3",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "910e1ca3-c2e6-4e54-b76a-a12c719aa8a3",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "code": "719303004",
+              "system": "http://snomed.info/sct",
+              "display": "Assessment of pregnancy status prior to imaging"
+            }
+          ],
+          "text": "Assessment of pregnancy status prior to imaging"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "effectiveDateTime": "2024-08-06T12:18:11+05:30",
+        "performer": [
+          {
+            "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+            "type": "Practitioner"
+          }
+        ],
+        "valueString": "Not pregnant"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:320b2b02-1872-484b-907f-f7d0d873fac0",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "320b2b02-1872-484b-907f-f7d0d873fac0",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "code": "80182007",
+              "system": "http://snomed.info/sct",
+              "display": "Irregular periods"
+            }
+          ],
+          "text": "Irregular periods"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "effectiveDateTime": "2024-08-06T12:18:11+05:30",
+        "performer": [
+          {
+            "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+            "type": "Practitioner"
+          }
+        ],
+        "valueString": "Irregular periods"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:1af9fa5c-28cb-4c45-8d0a-24002faddeb8",
+      "resource": {
+        "resourceType": "ServiceRequest",
+        "id": "1af9fa5c-28cb-4c45-8d0a-24002faddeb8",
+        "status": "active",
+        "intent": "order",
+        "code": {
+          "text": "Chest CT"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "occurrenceDateTime": "2013-05-08T09:33:27+07:00",
```

Comment:
how is it possible that the document is authored on 2024-08-06 and the occuranceDateTime (when the investigation should happen) be in the past? Put a future date, you dont need time.